git-hub

```diff
@@ -87,14 +97,142 @@ def push():
             other_user = data["user"]
             pr = data["pr"]
             branch = data["branch"]
-            p = subprocess.Popen(["git", "push", other_user,
-                                 f'{pr}:{branch}'], stdout=subprocess.PIPE)
+            p = subprocess.Popen(["git", "push", other_user, f'{pr}:{branch}'], stdout=subprocess.PIPE)
             p.stdout.read()
     except FileNotFoundError as e:
         print(e)
         print("This command must be used after git hub pr.")
         sys.exit(1)
 
+def search(keyword, user, comment, number, branch):
+    """Searches open and closed pull request comments for specified keyword.
+    Opens pull-requests.toml file in .git folder to fetch pull requests.
```

Comment:
Blank line after first line of docstring.  (1 line for a short description, then a second paragraph with longer description.)